doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/service-migration-con.adoc

```diff
@@ -3,6 +3,8 @@
 
 The process instance migration service is a utility for migrating process instances from one deployment to another. Process or task variables are not affected by the migration. However, the new deployment can use a different process definition.
 
+When migrating a process, the process instance migration service also automatically migrates all its subprocesses, subprocesses of its subprocesses, and so on. If you attempt to migrate a subprocess withput migrating the parent process, the migration fails.
```

Comment:
all its subprocesses, subprocesses of its subprocesses, and so on.

It's a little unclear what its is. Is it the process instance migration service, or is it a process? I'm guessing its the latter, and if so:
... all of the subprocesses that belong to the process, the subprocesses of those subprocesses, and so on.